src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6382,27 +6382,28 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
                 }
                 else
                 {    // Check Party/Raid Group
-                    if (Group* group = ToPlayer()->GetGroup())
```

Comment:
did the old code trigger a NULL dereference exception at this point when IsPet() returned true ?